dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAIChatCompletion_NoneFunctionChoiceBehaviorTests.cs

```diff
@@ -0,0 +1,219 @@
+    //    {
+    //        invokedFunctions.Add(context.Function.Name);
+    //        await next(context);
+    //    });
+
+    //    var promptTemplate = """"
+    //        template_format: semantic-kernel
+    //        template: How many days until Christmas?
+    //        execution_settings:
+    //          default:
+    //            temperature: 0.1
+    //            function_choice_behavior:
+    //              type: none
+    //        """";
+
+    //    var promptFunction = KernelFunctionYaml.FromPromptYaml(promptTemplate);
+
+    //    // Act
+    //    var result = await this._kernel.InvokeAsync(promptFunction);
+
+    //    // Assert
+    //    Assert.NotNull(result);
+
+    //    Assert.Empty(invokedFunctions);
+    //}
+
+    [Fact]
+    public async Task SpecifiedInCodeInstructsConnectorNotToInvokeKernelFunctionForStreamingAsync()
+    {
+        // Arrange
+        var plugin = this._kernel.CreatePluginFromType<DateTimeUtils>();
+        this._kernel.Plugins.Add(plugin);
+
+        var invokedFunctions = new List<string>();
+
+        this._autoFunctionInvocationFilter.RegisterFunctionInvocationHandler(async (context, next) =>
+        {
+            invokedFunctions.Add(context.Function.Name);
+            await next(context);
+        });
+
+        var settings = new OpenAIPromptExecutionSettings { FunctionChoiceBehavior = FunctionChoiceBehavior.None() };
+
+        string result = "";
+
+        // Act
+        await foreach (string c in this._kernel.InvokePromptStreamingAsync<string>("How many days until Christmas?", new(settings)))
+        {
+            result += c;
+        }
+
+        // Assert
+        Assert.NotNull(result);
+
+        Assert.Empty(invokedFunctions);
+    }
+
+    //[Fact]
+    //public async Task SpecifiedInPromptInstructsConnectorNotToInvokeKernelFunctionForStreamingAsync()
+    //{
+    //    // Arrange
+    //    this._kernel.ImportPluginFromType<DateTimeUtils>();
+
+    //    var invokedFunctions = new List<string>();
+
+    //    this._autoFunctionInvocationFilter.RegisterFunctionInvocationHandler(async (context, next) =>
+    //    {
+    //        invokedFunctions.Add(context.Function.Name);
+    //        await next(context);
+    //    });
+
+    //    var promptTemplate = """"
+    //        template_format: semantic-kernel
+    //        template: How many days until Christmas?
+    //        execution_settings:
+    //          default:
+    //            temperature: 0.1
+    //            function_choice_behavior:
+    //              type: none
+    //        """";
+
+    //    var promptFunction = KernelFunctionYaml.FromPromptYaml(promptTemplate);
+
+    //    string result = "";
+
+    //    // Act
+    //    await foreach (string c in promptFunction.InvokeStreamingAsync<string>(this._kernel))
+    //    {
+    //        result += c;
+    //    }
+
+    //    // Assert
+    //    Assert.NotNull(result);
+
+    //    Assert.Empty(invokedFunctions);
+    //}
```

Comment:
Will this be uncommented in a follow up PR?

Reply:
Yes, this test and a few others that are commented out will be uncommented in one of the following PRs.